lib/twitter/twitter_api.dart

```diff
@@ -32,6 +32,16 @@ class TwitterApi {
       throw Exception(responseJSON['errors'][0]['detail']);
     }
 
-    return TwitterUser.fromJson(responseJSON['data'] as Map<String, dynamic>);
+    final user = responseJSON['data'] as Map<String, dynamic>;
+
+    try {
+      if (responseJSON['includes'] != null) {
+        user['pinnedTweet'] = responseJSON['includes']['tweets'][0]['text'];
```

Comment:
Better move this to the `TwitterUser` object and take the list of tweets (new object) and search in all of them via a map or forEach